english/search-for-dom-cobb/search-for-dom-cobb.md

```diff
@@ -0,0 +1,4 @@
+You realize you came as a 'tourist' into Dom Cob's dream! Why does he have marshmallows for walls?
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.